frontend/src/interfaces/uiConfig.ts

```diff
@@ -48,6 +48,7 @@ export interface IFlags {
     proPlanAutoCharge?: boolean;
     notifications?: boolean;
     loginHistory?: boolean;
+    projectScopedSegments?: boolean;
```

Comment:
Is there any reason why this is optional? I think it can be made non optional without any compilation issue